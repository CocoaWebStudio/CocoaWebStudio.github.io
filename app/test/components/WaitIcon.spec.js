import { shallowMount } from '@vue/test-utils'
import WaitIcon from '~/components/WaitIcon/WaitIcon.vue'

describe('NavMenu component', () => {
  let cmp
  
  beforeEach(() => {
    cmp = shallowMount(WaitIcon, {
      props: {
        show:true
      }
    })
  })
  
  it('is a Vue instance', () => {
    // Within cmp.vm, we can access all Vue instance methods
    expect(cmp.isVueInstance()).toBeTruthy()
  })
  
  it("has the expected html structure", () => {
    expect(cmp.element).toMatchSnapshot()
  })
})
