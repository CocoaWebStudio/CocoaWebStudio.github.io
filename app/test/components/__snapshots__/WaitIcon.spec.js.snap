// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavMenu component has the expected html structure 1`] = `<!---->`;
